azure-go-aks-helm/config.go

```diff
@@ -30,7 +30,7 @@ func configure(ctx *pulumi.Context) (Config, error) {
 
 	out.K8sVersion = cfg.Get("k8sVersion")
 	if out.K8sVersion == "" {
-		out.K8sVersion = "1.18.14"
```

Comment:
This k8s version was no longer supported. This broke compatibility with
[lines 57-61 above in cluster.go](https://github.com/pulumi/examples/pull/1154/files#diff-be3b69ab145b219a5fad311278e9587a1f75f7976415a6ccac67d99718d2ffd9L57-L61)
so I removed it. It didn't appear to be a significant part of the README.

Reply:
Sounds good.